distributions/aws/examples/rds-s3/README.md

```diff
@@ -63,6 +63,25 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
 
 Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. 
 
+6. Install AWS Secrets & Configuration Provider with Kubernetes Secrets Store CSI driver
+
+Run the following commands to enable oidc and create an iamserviceaccount with permissions to retrieve secrets from AWS Secrets Manager
+
+```
+eksctl utils associate-iam-oidc-provider --region=$CLUSTER_REGION --cluster=$CLUSTER_NAME --approve
+
+kubectl create namespace kubeflow
+
+eksctl create iamserviceaccount  --name kubeflow-secrets-manager-sa  --namespace kubeflow  --cluster $CLUSTER_NAME --attach-policy-arn \ arn:aws:iam::aws:policy/AmazonSSMReadOnlyAccess --attach-policy-arn arn:aws:iam::aws:policy/SecretsManagerReadWrite --override-existing-serviceaccounts   --approve --region $CLUSTER_REGION
+```
+
+Run these commands to install AWS Secrets & Configuration Provider with Kubernetes Secrets Store CSI driver
+
+```
+helm repo add secrets-store-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts
+helm -n kube-system install csi-secrets-store secrets-store-csi-driver/secrets-store-csi-driver  --set syncSecret.enabled=true
+curl -s https://raw.githubusercontent.com/aws/secrets-store-csi-driver-provider-aws/main/deployment/aws-provider-installer.yaml | kubectl apply -f - 
```

Comment:
Is there an SDK for this or is CLI the only option? Asking for testing.

Reply:
do you mean an sdk to create the secrets rather than the cli command I provided?

Reply:
On another note...can we stick to `kubectl` only ?
If not then you need to update pre-requisite to add `helm`.

TBH kubeflow community does not recommend to use helm...so sort of avoid any bad ux, lets stick to kubectl only.